crypto/BRCryptoCipher.h

```diff
@@ -54,16 +54,23 @@ extern "C" {
 
     extern size_t
     cryptoCipherDecryptLength (BRCryptoCipher cipher,
-                               const char *ciphertext,
+                               const uint8_t *ciphertext,
                                size_t ciphertextLen);
 
     extern BRCryptoBoolean
     cryptoCipherDecrypt (BRCryptoCipher cipher,
                          uint8_t *plaintext,
                          size_t plaintextLen,
-                         const char *ciphertext,
+                         const uint8_t *ciphertext,
```

Comment:
Oversight here in initial commit (should have always been `uint8_t *`; copy-pasta from the BRCryptoCoder I suspect).